datasets/us/dod_chinese_milcorps/us_dod_chinese_milcorps.yaml

```diff
@@ -0,0 +1,21 @@
+name: us_dod_chinese_milcorps
+title: "Chinese military companies"
+url: https://www.defense.gov/News/Releases/Release/Article/3661985/dod-releases-list-of-peoples-republic-of-china-prc-military-companies-in-accord/
+
+description: >
+    A list of companies identified as Chinese military companies operating in the United States.
+    The list is expected to be maintained by the U.S. Department of Defense.
```

Comment:
The first line is perfect for the `summary` field which is presented almost as a pullout on the website.

For the `description` field, I think we can expand a bit in areas like

- how frequently is it updated
- reference to the act and section (Section 1260H of the National Defense Authorization Act for Fiscal Year 2021)

You can probably quote the whole middle paragraph with `>`s in the markdown at the end - perhaps the lowest priority info about it, but still interesting background

> Updating the Section 1260H list of "Chinese military companies" is an important continuing effort in highlighting and countering the PRC's Military-Civil Fusion strategy. The PRC's Military-Civil Fusion strategy supports the modernization goals of the People's Liberation Army (PLA) by ensuring it can acquire advanced technologies and expertise developed by PRC companies, universities, and research programs that appear to be civilian entities. Section 1260H directs the Department to begin identifying, among other things, Military-Civil Fusion contributors operating directly or indirectly in the United States.

Reply:
Alright, I've added a bit about this and specifically called out that while DoD might publish more companies it won't automatically mean that the CSV file is updated as it was prepared by us.